.github/workflows/R-CMD-CHECK-release.yml

```diff
@@ -5,11 +5,12 @@ on:
     branches:
       - main
       - feature/*
+  pull_request:
 
```

Comment:
If this is generally working, I think we can get rid of this trigger agian.
```suggestion

```

Reply:
ah! yes. slightly gas-addled brain over here 🥴